src/fragments/start/getting-started/ionic/setup.mdx

```diff
@@ -15,7 +15,9 @@ This runs a development server and allows us to see the output generated by the
 
 ## Initialize a new backend
 
-Now that we have a running Ionic app, it's time to set up Amplify so that we can create the necessary backend services needed to support the app. From the root of the project, run:
+Now that we have a running Ionic app, it's time to set up Amplify so that we can create the necessary backend services needed to support the app. 
+
+Open a new terminal. From the root of the project, run:
```

Comment:
should this section be removed so as not to conflict with the Ionic PR to remove this file?

Reply:
Yes, Ionic should not be a part of this PR